scripts/check-breaking-changes.sh

```diff
@@ -4,7 +4,7 @@
 PACKAGES=$(ls -d ./packages/*/)
 EXIT_CODE=0
 GITHUB_MESSAGE=""
-SKIP_PACKAGES=("grafana-eslint-rules" "grafana-plugin-configs")
+SKIP_PACKAGES=("grafana-eslint-rules" "grafana-plugin-configs" "grafana-o11y-ds-frontend")
```

Comment:
Are you sure you wanna disable levitate on this package? Even though it's private it might help track down issues. 🤷

Reply:
Fair enough. This PR has highlighted that we're making a few assumptions in the levitate PR checks which causes failures. I'll try to get something on our board so we can remove this in the future.